### PR TITLE
keep keptAt is not optional

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

**Comment by eishay** [comment]
@andrewconner @stkem

File: kifi-backend/modules/shoebox/marketing [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 56906cc7c0733af9e06e415f3b7d88a3c1b894c6
+Subproject commit 17af4006a9f9347561fa4b118748a12c65c63afa
```
Comment (2is10) [inline kifi-backend/modules/shoebox/marketing]: unintentional?

**Comment by 2is10** [comment]
Extension still isn’t sending bookmark times, but I could add it today.
https://app.asana.com/0/2456298849186/20632537806440

File: kifi-backend/modules/shoebox/marketing (hunk 1 above)
Comment (eishay) [inline kifi-backend/modules/shoebox/marketing]: right

**Comment by eishay** [comment]
@2is10 that will be nice

**Comment by stkem** [comment]
:+1:

**Pull request merged**